src/main/java/org/apache/commons/collections4/map/DefaultedMap.java

```diff
@@ -110,7 +110,7 @@
      */
     public static <K, V> DefaultedMap<K, V> defaultedMap(final Map<K, V> map, final Factory<? extends V> factory) {
         if (factory == null) {
```

Comment:
Not quite, to go the NPE route, you should use a liner with `Objects.requireNonNull()`.

Reply:
Changed to use `Objects.requireNonNull` (I noticed and followed commit 513ae545ee2b333f62f8ff989e1da161d807d76d)